database/tbcd/level/level.go

```diff
@@ -108,16 +110,26 @@ func headerHash(header []byte) *chainhash.Hash {
 }
 
 type Config struct {
-	Home             string // home directory
-	BlockCache       int    // number of blocks to cache
-	BlockheaderCache int    // number of blocks headers to cache
+	Home                 string // home directory
+	BlockCache           int    // number of blocks to cache
+	BlockheaderCacheSize string // size of block header cache
+	blockheaderCacheSize int    // paser size of block header cache,
```

Comment:
nit: block header is two words, as such it could make sense to capitalise the H:
```suggestion
	BlockHeaderCacheSize string // size of block header cache
	blockHeaderCacheSize int    // paser size of block header cache,
```

Reply:
Nein. I have resisted that and will continue to resist that :-)